src/couch_replicator_manager.erl

```diff
@@ -213,11 +216,13 @@ handle_cast(Msg, State) ->
     twig:log(error, "Replication manager received unexpected cast ~p", [Msg]),
     {stop, {error, {unexpected_cast, Msg}}, State}.
 
-handle_info({nodeup, _Node}, State) ->
-    {noreply, rescan(State)};
+handle_info({nodeup, Node}, State) ->
+    Live = lists:usort([Node|State#state.live]),
```

Comment:
I'm suddenly wondering if we shouldn't just use `[node() | nodes()]` every time we get an up/down message. I can't immediately think of a way to get the addition/subtraction to get out of whack, but we could overcome any weird issues if we just grabbed the whole list each time.